packages/framework/src/Resources/translations/messages.cs.po

```diff
@@ -2215,8 +2233,8 @@ msgstr "Přepočítávání cen bylo zakázáno, nezapomeňte upravovat všechny
 msgid "VAT"
 msgstr "DPH"
 
-msgid "VAT <strong>{{ name }}</strong> deleted"
-msgstr "DPH <strong>{{ name }}</strong> bylo smazáno"
+msgid "VAT <strong>%name%</strong> deleted"
+msgstr "DPH <strong>%name%</strong> bylo smazáno"
```

Comment:
This one should stay as it was. Editing only the `msgstr` should be enough and you will prevent BC breaks.

Reply:
fixed in 4a6f7ac